pages/404html.js

```diff
@@ -0,0 +1,68 @@
+const error404 = `<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Blogium-Service / Not found route</title>
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,700">
+	<style>
+		body {
+			background: #0e141a;
+			font-family: 'Open Sans', sans-serif;
```

Comment:
If I may, I'd suggest using system fonts instead of downloading a resource. There's a robust alternative that's used by Github, Zeit and other big companies:
```css
font-family: -apple-system,
  BlinkMacSystemFont,
  "Segoe UI",
  Roboto,
  Helvetica,
  Arial,
  sans-serif,
  "Apple Color Emoji",
  "Segoe UI Emoji",
  "Segoe UI Symbol";
```
saves a render-blocking and provides the page with the look&feel of the system where it's being showed.

Let me know what you think about that.